### PR TITLE
Trying to fix issue #6

### PR DESCRIPTION
**Pull request by baterflyrity**
Added arbitrary order of received from OS files and folders sorting to avoid different hashes for each OS.

### CODE REVIEW

File: checksumdir/__init__.py [hunk 1]
```diff
@@ -46,6 +46,9 @@ def dirhash(dirname, hashfunc='md5', excluded_files=None, ignore_hidden=False,
         if ignore_hidden and re.search(r'/\.', root):
             continue
 
+		dirs.sort()
```
Comment (Mochnant) [inline checksumdir/__init__.py]: This indentation looks too deep.  Shouldn't it be under the `for` loop?

Comment (baterflyrity) [reply checksumdir/__init__.py]: Yap it should. I don't know how to use github so i've fixed it in repo. Is it OK now?

> This indentation looks too deep. Shouldn't it be under the `for` loop?

Comment (Mochnant) [reply checksumdir/__init__.py]: It actually looks like it is still off by 1 space.  Has this code been tested to work as expected?

Comment (baterflyrity) [reply checksumdir/__init__.py]: > It actually looks like it is still off by 1 space. Has this code been tested to work as expected?

On my PC it works fine. Also in my fork and in review here it's fine. Maybe you look at outdated version or pull request doesn't work properly?

Comment (Mochnant) [reply checksumdir/__init__.py]: We can leave it to @cakepietoast to make a final determination, but here's what I see:

Screen Shot 2019-07-24 at 2 55 23 PM

Comment (baterflyrity) [reply checksumdir/__init__.py]: That's fine. Under for loop. As it must be.

Comment (Mochnant) [reply checksumdir/__init__.py]: If you look closer you will see that it is not indented as it should be.  It is one space closer than the others.  That will throw an `IndentationError` in Python.

Comment (baterflyrity) [reply checksumdir/__init__.py]: > If you look closer you will see that it is not indented as it should be. It is one space closer that the others. That will throw an `IndentationError` in Python.

I repush to be sure.

Comment (Mochnant) [reply checksumdir/__init__.py]: That looks correct now, thank you!

**Comment by Mochnant** [comment]
You might want to squash those commits to 1, as well.

**Comment by baterflyrity** [comment]
> You might want to squash those commits to 1, as well.

I don't know how to work with github.

**Comment by Mochnant** [comment]
No problem, @cakepietoast can squash and merge if they like.

**Comment by to-mc** [comment]
Appreciate the PR, will check it and merge as soon as I can!

**Comment by baterflyrity** [comment]
@Mochnant So... I guest you might test and close issue #6 .

**Pull request merged**